uniprotkb-rest/src/test/java/org/uniprot/api/uniprotkb/controller/UniProtKBSearchControllerIT.java

```diff
@@ -1700,7 +1714,11 @@ protected SearchParameter searchQueryWithInvalidValueQueryReturnBadRequestParame
                                             "The 'structure_3d' filter value can only be true or false",
                                             "The 'taxonomy_id' filter value should be a number",
                                             "The 'accession_id' filter value 'INVALID' has invalid format. It should be a valid UniProtKB accession",
-                                            "The 'virus_host_id' filter value should be a number")))
+                                            "The 'virus_host_id' filter value should be a number",
+                                            "The 'uniparc' filter value has invalid format. It should be a valid UniParc upi.",
+                                            "The 'uniref_cluster_50' filter value has invalid format. It should be a valid UniRef50 cluster id.",
+                                            "The 'uniref_cluster_90' filter value has invalid format. It should be a valid UniRef90 cluster id.",
+                                            "The 'uniref_cluster_100' filter value has invalid format. It should be a valid UniRef100 cluster id.")))
```

Comment:
id -> ID (i didn't realise this, but i just looked it up. apparently we should make id capitalised, to ID)